CHANGES.md

```diff
@@ -48,6 +48,7 @@ Released on June 28, 2023.
  -  Added `IBlockChainStates.GetState()` interface method.  [[#3250]]
  -  Added `IBlockStates.GetState()` interface method.  [[#3250]]
  -  Changed `IBlockStates` to inherit `IAccountState` interface.  [[#3251]]
+ -  Added `BlockType.PreEvaluationHash` field. [[#3281]]
```

Comment:
I guess this pull request's branch is not on the latest commit of the `main` branch. Could you rebase the branch on the `main` branch?